zero_to_hero_guide/quickstart.md

```diff
@@ -82,20 +102,35 @@ If you're looking for more specific topics like tool calling or agent setup, we
      llama stack build --template ollama --image-type conda
      ```
 
-2. **Edit Configuration**:
-   - Modify the `ollama-run.yaml` file located at `/Users/yourusername/.llama/distributions/llamastack-ollama/ollama-run.yaml`:
-     - Change the `chromadb` port to `8000`.
-     - Remove the `pgvector` section if present.
+After this step, you will see the console output:
+
+```
+Build Successful! Next steps: 
+   1. Set the environment variables: LLAMASTACK_PORT, OLLAMA_URL, INFERENCE_MODEL, SAFETY_MODEL
+   2. `llama stack run /Users/username/.llama/distributions/llamastack-ollama/ollama-run.yaml`
+```
+
+2. **Set the ENV variables by exporting them to the terminal**:
+```bash
+export OLLAMA_URL="http://localhost:11434"
+export LLAMA_STACK_PORT=5001
+export INFERENCE_MODEL="meta-llama/Llama-3.2-3B-Instruct"
+export SAFETY_MODEL="meta-llama/Llama-Guard-3-1B"
+```
 
 3. **Run the Llama Stack**:
-   - Run the stack with the configured YAML file:
+   - Run the stack with command shared by the API from earlier:
      ```bash
-     llama stack run /path/to/your/distro/llamastack-ollama/ollama-run.yaml --port 5050
+     llama stack run /Users/username/.llama/distributions/llamastack-ollama/ollama-run.yaml  \
```

Comment:
you can simply do `llama stack run ollama` and that would be sufficient I think

Reply:
thanks! made all the necessary changes